integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -108,8 +111,10 @@ private static void recoverGravitinoServerConfig() throws IOException {
   protected static void downLoadMySQLDriver(String relativeDeployLibsPath) throws IOException {
     if (!ITUtils.EMBEDDED_TEST_MODE.equals(testMode)) {
       String gravitinoHome = System.getenv("GRAVITINO_HOME");
-      java.nio.file.Path tmpPath = Paths.get(gravitinoHome, relativeDeployLibsPath);
-      JdbcDriverDownloader.downloadJdbcDriver(DOWNLOAD_JDBC_DRIVER_URL, tmpPath.toString());
+      Path tmpPath = Paths.get(gravitinoHome, relativeDeployLibsPath);
+      JdbcDriverDownloader.downloadJdbcDriver(DOWNLOAD_MYSQL_JDBC_DRIVER_URL, tmpPath.toString());
+      JdbcDriverDownloader.downloadJdbcDriver(
```

Comment:
You should remove other download actions in the sub-classes

Reply:
Ok